pkg/util/util.go

```diff
@@ -90,6 +90,10 @@ func GetClusterIDFromCableName(cableName string) string {
 	// 0           1    2   3    4    5       6   7  8 9  10
 	// submariner-cable-my-super-long_cluster-id-172-16-32-5
 	cableSplit := strings.Split(cableName, "-")
+	if len(cableSplit) != 11{
+		klog.Error(fmt.Errorf("cableName:%s format is invalid",cableName))
+		return ""
+	}
```

Comment:
What is the motivation for this change? It is actually incorrect - the token length does not need to be 11 - the comment above is just an example of one that is long.

Reply:
Sure - we expect the length to be at least 3. 

This really shouldn't be an issue unless someone purposely tampers with it. Are you seeing an issue or how did this code come to your attention?

Reply:
I'am reading the code and  found it inadvertently

Reply:
Thanks for the context @tpantelis, that makes sense.